cfgov/v1/atomic_elements/organisms.py

```diff
@@ -170,6 +170,46 @@ class Meta:
         label = ' '
 
 
+class BureaStructurePosition(blocks.StructBlock):
+    name = blocks.CharBlock()
+    title = blocks.StructBlock([
+        ('line_1', blocks.CharBlock(required=False)),
+        ('line_2', blocks.CharBlock(required=False))
+    ])
+
+
+class BureaStructureNode(BureaStructurePosition):
+    office_title = blocks.CharBlock(required=False)
+    positions = blocks.ListBlock(BureaStructurePosition(required=False))
+
+
+class BureaStructureOffice(blocks.StructBlock):
+    name = blocks.CharBlock()
+    title = blocks.StructBlock([
+        ('line_1', blocks.CharBlock(required=False)),
+        ('line_2', blocks.CharBlock(required=False))
+    ])
+    office_title = blocks.CharBlock(required=False)
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2532/commits/861674d42470d1ebe15c3ac204e66b4fa58f741c.